x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/data_view_prompt/data_view_prompt.tsx

```diff
@@ -24,43 +24,45 @@ export const DataViewPrompt: FC<Props> = ({ destIndex, color }) => {
     },
   } = useMlKibana();
 
+  const {
+    euiTheme: { size },
+  } = useEuiTheme();
+
   const canCreateDataView = useMemo(
     () =>
       capabilities.savedObjectsManagement.edit === true || capabilities.indexPatterns.save === true,
     [capabilities]
   );
 
   return (
-    <>
-      <EuiText size="xs" color={color ?? 'warning'}>
+    <EuiText size="xs" color={color ?? 'warning'} css={{ padding: `${size.s}` }}>
+      <FormattedMessage
+        id="xpack.ml.dataframe.analytics.dataViewPromptMessage"
+        defaultMessage="No data view exists for index {destIndex}. "
+        values={{
+          destIndex: destIndex ?? '',
+        }}
+      />
+      {canCreateDataView === true ? (
         <FormattedMessage
-          id="xpack.ml.dataframe.analytics.dataViewPromptMessage"
-          defaultMessage="No data view exists for index {destIndex}. "
+          id="xpack.ml.dataframe.analytics.dataViewPromptLink"
+          defaultMessage="{linkToDataViewManagement}{destIndex}."
```

Comment:
I'm not sure why some of these are showing up as changes - the only changes were removing the unnecessary wrapper <></> and adding the padding 🤔 
The rest of the code is the same on main. Will look into it.